.changeset/large-dolls-dress.md

```
@@ -0,0 +1,8 @@
+---
+'@mastra/platform-workspace': patch
+'@mastra/factory': patch
+'@mastra/code-sdk': patch
+'@mastra/core': patch
+---
+
+Send opaque acting-user subjects with Platform sandbox requests, including Factory creation and reattachment flows.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)